owlbot.py

```diff
@@ -17,19 +17,31 @@
 # This code snippet can be used to manually update the typeless bot
 # to a different version than OwlBot has baked in, but this shouldn't
 # be needed once it's settled down.
-"""
 import os
 from synthtool import shell
 from synthtool.log import logger
 old_path = os.getcwd()
 os.chdir("/synthtool")
-logger.debug("Update typeless sample bot [1.1.0]")
-shell.run(["npm", "i", "@google-cloud/typeless-sample-bot@1.1.0"])
+logger.debug("Update typeless sample bot [1.3.0]")
+shell.run(["npm", "i", "@google-cloud/typeless-sample-bot@1.3.0"])
 os.chdir(old_path)
-"""
 
 node.typeless_samples_hermetic()
 
+# We need to run this before the main owlbot processing, to make
+# sure quickstart.js gets gts fixed before the README is generated.
+# This needs to be worked out more properly, this is temporary.
+logger.debug("Copy eslint config")
+shell.run(
```

Comment:
`gts` should already be built into the container, see:

https://github.com/googleapis/synthtool/blob/master/synthtool/languages/node.py#L216

Which should be called by `owlbot_main` already.

Reply:
Yeah, the trouble is that there's a chicken and egg problem that needs solving here :\ We need to gts fix the quickstart.js before the README gets generated by OwlBot. There are probably better ways to fix this, I'm happy to talk it over :)

Reply:
I wonder if we could change the order of operations in `synthtool` to solve this?

Reply:
Mentioned in chat, but recorded here for posterity: because of the README generation step, the quickstart requires a gts pre-pass, but I'll work out how to fix this and come back with a PR removing the manual install and hopefully fixing that.